### PR TITLE
Replace input in register

### LINKED ISSUE #140: Reafctor input in registration

### PR DESCRIPTION
**Pull request by Sagi-Viktor**
closes #140

### CODE REVIEW

File: angular/src/app/components/navigation-bar/navigation-bar.component.html [hunk 1]
```diff
@@ -4,4 +4,4 @@
   <button mat-flat-button type="button" routerLink="/league-posts">League Posts</button>
   <button mat-flat-button type="button" (click)="openRegisterModal()" >Registration!</button>
   <button mat-flat-button type="button" routerLink="/profile/1">Profile</button>
-</nav>
+</nav>Technologies
```
Comment (Zsezsu) [inline angular/src/app/components/navigation-bar/navigation-bar.component.html]: what is this?

Comment (Sagi-Viktor) [reply angular/src/app/components/navigation-bar/navigation-bar.component.html]: Ops

File: angular/src/app/components/user/register/user-register-dialog.component.html [hunk 2]
```diff
@@ -23,8 +23,11 @@ <h1 mat-dialog-title align="center">Registration</h1>
             data.email != emailForMach">Provided emails are different</mat-hint>
   </mat-form-field>
   <mat-form-field appearance="fill">
-    <mat-label>Password</mat-label>
-    <textarea class="password-input" matInput [(ngModel)]="data.password"></textarea>
+    <mat-label>Enter your password</mat-label>
+    <input matInput [type]="hide ? 'password' : 'text'" [(ngModel)]="data.password">
+    <button mat-icon-button matSuffix (click)="hide = !hide" [attr.aria-label]="'Hide password'" [attr.aria-pressed]="hide">
+      <mat-icon>{{hide ? 'visibility_off' : 'visibility'}}</mat-icon>
+    </button>
```
Comment (Zsezsu) [inline angular/src/app/components/user/register/user-register-dialog.component.html:30]: very nice password input!! :+1:

File: angular/src/app/services/league-post/league-post.service.ts [hunk 3]
```diff
@@ -29,12 +29,13 @@ export class LeaguePostService {
   }
 
   getLeaguePostById(id: number): Observable<LeaguePost> {
-    const url = `${this.leaguePostsUrl}/${id}`;
+    const url = `api/leaguePosts/${id}`;
```
Comment (Zsezsu) [inline angular/src/app/services/league-post/league-post.service.ts]: öööö?

**Review by Zsezsu — APPROVED** [review]
:+1:

**Pull request merged**